internal/pkg/skuba/kubeadm/configmap.go

```diff
@@ -62,6 +62,18 @@ func GetCurrentClusterVersion() (*version.Version, error) {
 	return version.MustParseSemantic(initCfg.KubernetesVersion), nil
 }
 
+// GetKubeadmApisVersion returns the api version to use in the kubeadm-init.conf
+func GetKubeadmApisVersion() (string, error) {
+	current, err := GetCurrentClusterVersion()
```

Comment:
seems we can't use that during bootstrap as there's no way to query the apiserver yet. i guess we have to get this from the init configuration instead

Reply:
updated